scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -321,75 +321,60 @@ public void onClick(ClickEvent clickEvent) {
      *                +-----------+
      * </pre>
      */
+    @SuppressWarnings("squid:S3776")
     private ToolStrip buildTools() {
         ToolStrip tools = new ToolStrip();
         tools.setHeight(50);
         tools.setWidth100();
         tools.setBackgroundImage("");
-        tools.setBackgroundColor("#fafafa");
+        tools.setBackgroundColor(logoStripBackgroundColor);
         tools.setBorder("0px");
 
         MenuItem submitMenuItem = new MenuItem("Submit job",
                                                SchedulerImages.instance.job_submit_16().getSafeUri().asString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)